### DISCUSSION TITLE
What is the difference to OpenBVE?

### DISCUSSION THREAD

**Discussion by Dinock** [Libre-TrainSim/Libre-TrainSim#18] [Q&A] [answered]
Hey Jean28518,
thank you very much for working on this interesting project. I am looking forward to playing the next update. :)

I just have a general question: Why did you start this project and what is the difference to [OpenBVE](https://github.com/leezer3/OpenBVE)? Why should I play and support Libre-TrainSim and not OpenBVE?

I don't want to sound nitpicking, I just don't really understand. (Maybe also because I am new to GitHub and train simulators.)

**Comment by Jean28518** [comment, marked as answer]
Hey Dinock,
great to hear!

I played a lot OpenBVE in the last years, and I like it in a specific way. But there are many causes why I started the Libre TrainSim Project:
- OpenBVE is old. Adding such things as real AI trains, or even a good usable track editor is very hard.
- Creating Tracks in OpenBVE is quite time consuming. Also building Rails which aren't the main rail 
 is very messy from the engine itself. (Also it doesn't render Rails, you have to create for every single curve a new object. Also tracks are added only 25 m wise. To add some more intuitive features I think you have to rewrite the engine.
- OpenBVE doesn't run natively on Linux, and has some dependencies. - GoDot runs out of the box on windows, mac, linux, and even in the webbrowser or mobile phones..
- OpenBVEs AddOn handling is in my eyes a mess. Every Add-On Creator has to host his stuff itself, there's no official AddOn Platform. There are many tracks/trains, which I have on my harddrive but can't publish them because I litteraly have no rights for them - the most tracks even don't have any license... An official hosted AddOn Platform which forces the creator to add at least one license would solve this problem, and (if the License allows it) these tracks could live further on, and could be updated, even though the initial creator "leaves" his addons.
- I love trains and simulators, for me its a good practice developing a open source project, collaborating with others, and I have very much fun at it.
- Literally there don't exists a completely free simulator which doesn't meet my requirements

Overall I don't want to badmouth the OpenBVE Project, make your own experience! Also I don't want to 'destroy' OpenBVE or something else. I just want to develop an alternative to OpenBVE with modern components, easier tools, better AddOn handling and hosting, and mabye much more ;)

I orient myself at OpenBVE, LokSim3D, Train Simulator, and TrainSim World. In the end it will be a mix of all in my eyes good components ;)

**Comment by Dinock** [comment]
Thank you very much for your in-depth reply! 
I'm curious about how Libre TrainSim will develop and change over time. It's always nice to see people working on free open-source games and projects. :)

**Comment by Jean28518** [comment]
I am very curious about that too ;)

**Comment by mszekiel** [comment]
@Jean28518  take a look also on this one: https://web.td2.info.pl/en/ 😃
